src/interfaces.jl

```diff
@@ -218,13 +218,16 @@ function (PlotType::Type{<: AbstractPlot{Typ}})(scene::SceneLike, attributes::At
     # PlotType
 
     FinalType = Combined{Typ, ArgTyp}
+
+    trans = pop!(attributes, :transformation, automatic)
+
     plot_attributes = merged_get!(
         ()-> default_theme(scene, FinalType),
-        plotsym(FinalType), scene, attributes
+        plotsym(FinalType), scene, attributes;
+        allowlist = Set([:text, :xautolimits, :yautolimits, :zautolimits, :interpolate_in_fragment_shader, :label, :fxaa, :rasterize, :model])
```

Comment:
`inspectable`, `material`, `texture`, `transformation`, are also things which should be added, plus all the GLMakie lighting stuff (specular, diffuse, etc)

It would also be nice to have a signature which does not perform this check, for easy programmatic passing of attributes inherited from the parent plot!

Reply:
`transformation` is already removed elsewhere prior to this check

Wouldn't it be better to group all backend related keywords under one umbrella rather than just letting them potentially collide with the rest?

I think I would like it better if an RPRMakie plot had `meshscatter(data; RPRMakie = (; material = ...)`

Reply:
same for GLMakie

Reply:
If we want both name checking and the ability to add arbitrary attributes to a plot we need to separate them somehow. I was thinking we could have one whitelisted attribute Dict of core attributes, which may become fields of specific plot object (`struct Scatter ... end` etc) if we want that, and one Dict with everything else.

Having `RPRMakie = (; ...)` etc is similar to that I guess. Though I would say attributes that influence other objects (inspectable, label, ...) and overwrites (model, view, ...) should also move out of the directly defined attributes. I guess the latter could also be part of the backend groups. The former is kind of the "random stuff that you want to attach to a plot" group, which I think is useful to have in a dynamic way.

Reply:
we could also add a `misc` allowed key or something which should only ever be set to an `Attributes` instance with further keys. Then we could disallow such a key for any other recipe so we know it will never conflict, and people can use it to carry arbitrary stuff forward. I want to avoid a situation where people make recipes and we break them by introducing some new add-on keyword arg.